java/vector/src/main/codegen/templates/DenseUnionVector.java

```diff
@@ -812,12 +757,17 @@ public int getValueCount() {
   }
 
   public boolean isNull(int index) {
```

Comment:
This is inconsistent with getNullCount; this vector has nullCount == 0 but can have isNull return true regardless. C++ seems to always return false when there is no validity buffer.

Reply:
I am ok with changing this to always return false. However I think this means anyone using a Union vector would have to extract the child and check for null as an extra step. All other nested vectors have a top level validity buffer. I don't know what the ramifications of changing this are?

Reply:
I set `isNull` to always be false and all the tests seem to pass (with minor modifications). Take a look and let me know what you think.

Reply:
I think from the viewpoint of code generically manipulating vectors, this makes more sense - the entries of the union are never null, but they may reference something that is null. I agree it's less convenient when working with a union array explicitly though.